conventional-changelog-lighthouse/templates/commit.hbs

```diff
@@ -0,0 +1,46 @@
+* {{#if message ~}}
```

Comment:
license header for these templates, too (need to convert to handlebars comments, I guess)